plugin/pkg/auth/authorizer/rbac/rbac.go

```diff
@@ -116,7 +116,7 @@ func (r *RBACAuthorizer) Authorize(ctx context.Context, requestAttributes author
 			scope = "cluster-wide"
 		}
 
-		klog.Infof("RBAC DENY: user %q groups %q cannot %s %s", requestAttributes.GetUser().GetName(), requestAttributes.GetUser().GetGroups(), operation, scope)
+		klog.Infof("RBAC NO OPINION: user %q groups %q cannot %s %s", requestAttributes.GetUser().GetName(), requestAttributes.GetUser().GetGroups(), operation, scope)
```

Comment:
I'd suggest simplifying this to just `RBAC: ...`

Reply:
That would be OK with me.
I think that part of this PR shuld include changing the word "cannot" to something more like "not authorized to".
Maybe reword to something like: "RBAC: not authorizing user %q groups %q to %s %s"

Reply:
maybe `RBAC: no rules authorize user %q with groups %q to %s %s`

Reply:
that sounds good

Reply:
Sure, I'll push the required changes promptly.